src/computer.h

```diff
@@ -116,6 +116,8 @@ class computer
     private:
         // Difficulty of simply logging in
         int security;
+        // Date of next attempt
+        long next_attempt = 0;
```

Comment:
Why a `long`? You're storing an `int` here and you compare it to an `int`. I fear this might lead to compiler warnings (-> errors) on 64bit builds.